user-interface/src/data-verification/transfer/CaseSelection.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+function safeToInt(s: string) {
+  const intVal = parseInt(s);
+  if (isNaN(intVal)) return s;
+
+  return intVal.toString();
+}
+
+interface CaseSelectionAttributes {
+  courtDivisionName: string;
+  region: string;
+}
+
+interface CaseSelectionProps {
+  fromCourt: CaseSelectionAttributes;
+  toCourt: Partial<CaseSelectionAttributes>;
+}
+
+export function CaseSelection(props: CaseSelectionProps) {
```

Comment:
We do not appear to be using <CaseSelection> anymore. Delete?

Reply:
I believe we were planning to move this to the modal weren't we?